buildSrc/settings.gradle.kts

```diff
@@ -4,4 +4,10 @@ dependencyResolutionManagement {
             from(files("../gradle/libs.versions.toml"))
         }
     }
+
+    repositories {
```

Comment:
buildSrc still need separate repositories declaration, because it's a separate build; even if it's "included"